packages/fx-core/src/plugins/resource/bot/languageStrategy.ts

```diff
@@ -138,9 +138,13 @@ export class LanguageStrategy {
       //Typescript needs tsc build before deploy because of windows app server. other languages don"t need it.
       try {
         await utils.execute("npm install", packDir);
+      } catch (e) {
```

Comment:
I think it is OK to combine "npm install and npm run build" and ask user to try it again, so we don't need to separate them into two try...catch. What do you think?

Reply:
Both ok for me. Let's revert the change.